### DISCUSSION TITLE
AI x AI

### DISCUSSION THREAD

**Discussion by doehyunbaek** [prosyslab-classroom/cs524-program-analysis#428] [Art competition]
![_003b9850-ff5e-44a0-9f97-c8d83c2388d7](https://github.com/prosyslab-classroom/cs524-program-analysis/assets/70570804/b492082c-89df-464a-90e0-335ecb0a9a6f)

## Name

Doehyun Baek

## Tool You Used

[Bing Image Creator](https://www.bing.com/create)

## Short Description
For me, [*AI x AI*] means [*the human intelligence and the artificial intelligence striving together to approximate something infinite.*]".

## Long Description (up to 3 sentences)

There is no best abstraction of a circle in the polyhedra domain, but that doesn't mean we can improve our abstraction by adding one more constraint.
At this lonely struggle for striving for something infinite, a new friend of human intelligence has entered the battlefield, called artificial intelligence.
We shall see if they will lend a hand in drawing a better solution in the pursuit of better abstract interpretation.

**Comment by KihongHeo** [comment]
Very nice!
